docs/data/data-grid/style/style.md

```diff
@@ -127,15 +129,19 @@ import type {} from '@mui/x-data-grid/themeAugmentation';
 const theme = createTheme({
   mixins: {
     MuiDataGrid: {
-      // Pinned columns sections
-      pinnedBackground: '#340606',
-      // Headers, and top & bottom fixed rows
-      containerBackground: '#343434',
+      // Container background
+      background: '#fafaf9',
```

Comment:
How to handle dark mode here? It would be great to have this included in this demo.

Reply:
Mentioned here https://github.com/mui/mui-x/pull/16066#discussion_r1928401861